packages/docs/src/styles/pages/layout.scss

```diff
@@ -2,6 +2,10 @@ body {
   margin: 0;
 }
 
+:root {
```

Comment:
i think you need to include `:root, ::before, ::after, ::backdrop`. tried building this locally and couldn't see this change reflected. adding these additional pseudos helped (but why do we define them on all these different pseudos to begin with?).

i think because you didn't include these, your overwrite was being overwritten by them - if that makes sense

Reply:
ahh, weird. I didn't think the other pseudos applied to the scenario in which I was using the variable, so I didn't think it would be affected

Reply:
It works for me 😆 . I'll update it though

Reply:
Hmm, it doesn't work at all for me now. It must be the order in which stylesheets are loaded.

Reply:
i wondered about that as well.

Reply:
i'd think our docs site should load after any theme stuff though...